src/main/java/cs2103/v15_1j/jimjim/model/Event.java

```diff
@@ -7,50 +10,98 @@
 import javafx.beans.property.SimpleObjectProperty;
 
 public class Event extends TaskEvent implements Comparable<Event> {
-	private ObjectProperty<List<EventTime>> dateTimes;
+	private ObjectProperty<LocalDateTime> startDateTime;
+	private ObjectProperty<LocalDateTime> endDateTime;
 
 	public Event(String name, LocalDateTime start, LocalDateTime end) {
 		super(name);
-		this.dateTimes = new SimpleObjectProperty<List<EventTime>>(new ArrayList<EventTime>());
-		this.dateTimes.get().add(new EventTime(start, end));
+		this.setStartDateTime(start);
+		this.setEndDateTime(end);
 	}
 	
-	public Event(Event other) {
-		this(other.getName(), other.getEarliestDateTime(), other.getLatestDateTime());
+	public Event(Event e){
+		this(e.getName(), e.getStartDateTime(), e.getEndDateTime());
 	}
 
-	public List<EventTime> getDateTimes() {
-		return this.dateTimes.get();
+	public LocalDateTime getStartDateTime() {
+		return startDateTime.get();
 	}
 
-	public void addDateTime(LocalDateTime start, LocalDateTime end){
-		dateTimes.get().add(new EventTime(start, end));
+	public void setStartDate(LocalDate startDate){
+		LocalDateTime temp = LocalDateTime.of(startDate, startDateTime.get().toLocalTime());
+		this.startDateTime = new SimpleObjectProperty<LocalDateTime>(temp);
```

Comment:
those 2 lines can just be `this.startDateTime = ... (this.startDateTime.get().with(startDate)`